File: utils/supermarkets.py
from enum import Enum


class UKSupermarkets(Enum):
    TESCO = "TESCO"
    ASDA = "ASDA"
    SAINSBURYS = "SAINSBURYS"
    MORRISONS = "MORRISONS"
    WAITROSE = "WAITROSE"
    ALDI = "ALDI"
    LIDL = "LIDL"
    CO_OP = "CO-OP"

def supermarket_prefix(supermarket_enum):
    if type(supermarket_enum) is UKSupermarkets:
        return supermarket_enum.value + "_"
    else:
        return ""

File: crawlee-scraper/src/utils/supermarkets.js

const SupermarketType = {
    TESCO :"TESCO",
    ASDA :"ASDA",
    SAINSBURYS :"SAINSBURYS",
    MORRISONS :"MORRISONS",
    WAITROSE :"WAITROSE",
    ALDI :"ALDI",
    LIDL :"LIDL",
    CO_OP :"CO-OP"
}

const supermarketPrefix = (supermarket_enum) => {
    return supermarket_enum + "_"
}

export {
    SupermarketType,
    supermarketPrefix
}